pom.xml

```diff
@@ -59,15 +59,16 @@
         <com.googlecode.gson.version>2.10.1</com.googlecode.gson.version>
         <com.h2database.version>2.2.224</com.h2database.version>
         <com.jcraft.jsch.version>0.1.55</com.jcraft.jsch.version>
-        <com.squareup.okhttp3.version>4.12.0</com.squareup.okhttp3.version>
+        <com.squareup.okhttp3.version>5.0.0-alpha.12</com.squareup.okhttp3.version>
         <com.squareup.okio.version>3.6.0</com.squareup.okio.version>
         <commons-fileupload.version>1.5</commons-fileupload.version>
         <commons-io.version>2.14.0</commons-io.version>
         <commons-lang.version>2.6</commons-lang.version>
         <enforcer.skip>false</enforcer.skip>
         <integration.mysql.db.image>quay.io/eclipse/che--centos--mysql-57-centos7:latest-e08ee4d43b7356607685b69bde6335e27cf20c020f345b6c6c59400183882764</integration.mysql.db.image>
         <integration.postgresql.db.image>quay.io/eclipse/che--centos--postgresql-13-centos7:1-71b24684d64da46f960682cc4216222a7e4ed8b1a31dd5a865b3e71afdea20d2</integration.postgresql.db.image>
-        <io.fabric8.kubernetes-client>6.10.0</io.fabric8.kubernetes-client>
+        <io.fabric8.kubernetes-client.version>7.1.0</io.fabric8.kubernetes-client.version>
+        <io.fabric8.openshift-server-mock.version>6.13.5</io.fabric8.openshift-server-mock.version>
```

Comment:
You should remove this.
OpenShift Mock Server is not needed, you should switch to the vanilla Kubernetes Mock Server.
If you have trouble migrating, give me an example file and I'll show you the exact replacements you need to perform.

Reply:
done